### PR TITLE
Covid meds update

### PR DESCRIPTION
**Pull request by HelenCEBM**

### CODE REVIEW

File: notebooks/covid_meds/covid_meds.Rmd [hunk 1]
```diff
@@ -386,7 +388,7 @@ df_scmd_ddd <- df_scmd_mg %>%
 
 temp_ggplot <- df_scmd_ddd %>%
   group_by(year_month, vtmnm) %>%
-  summarise(volume_ddd = sum(volume_ddd)) %>% 
+  summarise(volume_ddd = sum(volume_ddd, na.rm = TRUE)) %>%
```
Comment (milanwiedemann) [inline notebooks/covid_meds/covid_meds.Rmd:391]: thannks for spotting this!

File: notebooks/covid_meds/covid_meds.Rmd [hunk 2]
```diff
@@ -468,7 +470,7 @@ temp_ggplot <- df_scmd_ddd_map %>%
   select(year_month, ods_code, vtmnm, volume_ddd, comm_region_name) %>% 
   mutate(comm_region_name = fct_explicit_na(comm_region_name)) %>% 
   group_by(comm_region_name, vtmnm) %>%
-  summarise(volume_ddd = sum(volume_ddd)) %>%
+  summarise(volume_ddd = sum(volume_ddd, na.rm = TRUE)) %>%
   group_by(comm_region_name) %>%
   mutate(prop_use = volume_ddd / sum(volume_ddd),
```
Comment (milanwiedemann) [inline notebooks/covid_meds/covid_meds.Rmd]: would also change this to `sum(volume_ddd, na.rm = TRUE)` or was there a particular reasons not to?

File: notebooks/covid_meds/covid_meds.Rmd [hunk 3]
```diff
@@ -185,6 +185,8 @@ df_scmd_names <- df_scmd %>%
            ods_code == "R1E" ~ "QNC", # Staffs
            ods_code == "TAD" ~ "QWO", # W Yorks
            ods_code == "TAJ" ~ "QUA", # Black country
+           ods_code == "TAH" ~ "QF7", # South Yorkshire & Bassetlow
+           ods_code == "TAF" ~ "QMJ", # North Central London
```
Comment (milanwiedemann) [inline notebooks/covid_meds/covid_meds.Rmd:189]: thanks! where did you get these codes from?

**Review by milanwiedemann — CHANGES_REQUESTED** [review]
I'd add `na.rm = TRUE` to the sum calculation in line 475 as well

File: notebooks/covid_meds/covid_meds.Rmd (hunk 3 above)
Comment (HelenCEBM) [reply notebooks/covid_meds/covid_meds.Rmd:189]: I created a list of trusts and their STPs to see if any are missing STPs. Then just google e.g. "NHS Trust TAF" to find the name (though we can also find this in the trust file supplied alongside the scmd file). Normally they have a name like "NHS Sheffield X Trust" and then I check to see what STP other Sheffield Trusts are in.

File: notebooks/covid_meds/covid_meds.Rmd (hunk 2 above)
Comment (HelenCEBM) [reply notebooks/covid_meds/covid_meds.Rmd]: Probably just missed this one, will fix, thanks!

File: notebooks/covid_meds/covid_meds.Rmd (hunk 3 above)
Comment (milanwiedemann) [reply notebooks/covid_meds/covid_meds.Rmd:189]: thx!

**Pull request merged**